src/wordpress-recommendations.lib.php

```diff
@@ -26,70 +25,244 @@
+                __('Admin/Administrator username still exists', 'sucuri-scanner') => __('Using an unique username and removing the default admin/administrator account make it more difficult for attackers to brute force your WordPress.', 'sucuri-scanner'),
+            );
+            // phpcs:enable
+        }
+
+        /*
+         * Check if user is using the super-admin for day-to-day operations.
+         * @see https://sucuri.net/guides/wordpress-security/#uac
+         */
+        $wpUsersCount = count_users();
+        if ($wpUsersCount['total_users'] === 1) {
+            // phpcs:disable Generic.Files.LineLength
+            $recommendations['lonelySuperAdmin'] = array(
+                __('Use super admin account only when needed', 'sucuri-scanner') => __('Create an Editor account instead of always using the super-admin to reduce the damage in case of session hijacking.', 'sucuri-scanner'),
+            );
+            // phpcs:enable
+        }
+
+        /*
+         * Check if user a 2FA plugin is being used.
+         * @see https://sucuri.net/guides/wordpress-security/#pwd
+         *
+         * First the script will get the plugins name, then run a regex match to
+         * identify possible 2FA plugins being used.
+         *
+         * NOTE: $wpPluginsInstalledName, $wpPluginsActivatedName, $wpPluginsDeactivatedName
+         * are created by this feature.
+         */
+        $wpPluginsInstalled = get_plugins();
+        foreach ($wpPluginsInstalled as $pluginPath => $pluginDetails) {
+            $wpPluginsInstalledName[] = $pluginDetails['Name'];
+            if (is_plugin_active($pluginPath)) {
+                $wpPluginsActivatedName[] = $pluginDetails['Name'];
+            } else {
+                $wpPluginsDeactivatedName[] = $pluginDetails['Name'];
+            }
+        }
+        // phpcs:disable Generic.Files.LineLength
+        $wp2faPluginsInstalled = preg_grep('/(2FA|2\sFactor|Two\sFactor|Two-Factor|2-Factor|Secure\sLogin|SecSign|Authentication|Wordfence\sSecurity|iThemes\sSecurity\sPro|Limit\sLogin|LDAP)/i', $wpPluginsInstalledName);
+        if (empty($wp2faPluginsInstalled)) {
+            $recommendations['loginUnprotected'] = array(
+                __('Install a 2FA plugin', 'sucuri-scanner') => __('2FA protects your account in the event that someone is able to guess/crack your password.', 'sucuri-scanner'),
+            );
+        }
+        // phpcs:enable
+
+        /*
+         * Check for unwanted extensions.
+         * @see https://sucuri.net/guides/wordpress-security/#apt
+         *
+         * Triggered if there are more than 2 themes or 5 plugins disabled and the
+         * WordPress install is not a multisite.
+         *
+         * NOTE: This check uses $wpPluginsDeactivatedName which is created by
+         * the 2FA plugin check.
+         */
+        if ((count(wp_get_themes()) > 2) || (count($wpPluginsDeactivatedName) > 5) && !is_multisite()) {
```

Comment:
Check the parenthesis. Currently it's `(A) or (B) and C` but you probably meant `(A or B) and C`.

Reply:
I kinda like the idea of showing the recommendation even if there's only 1 disabled plugin (instead of 5). @yaelcampo @ntorgagd would love to know what you think!